optaplanner-benchmark/src/test/java/org/optaplanner/benchmark/impl/ranking/ResilientScoreComparatorTest.java

```diff
@@ -24,7 +24,7 @@
 import org.optaplanner.core.api.score.Score;
 import org.optaplanner.core.api.score.buildin.hardsoft.HardSoftScore;
 import org.optaplanner.core.api.score.buildin.simple.SimpleScore;
-import org.optaplanner.core.impl.score.buildin.simple.SimpleScoreDefinition;
+import org.optaplanner.core.impl.score.buildin.SimpleScoreDefinition;
```

Comment:
Nitpick: Maybe I wouldn't drop the package "simple" etc of the score definition packages, because we can't drop them of the score API ones - just to keep them consistent. On the other hand, for 9.x it would make sense to consider dropping them of the score API ones too.

Reply:
Ok, nevermind this, I just saw "director.drools.holder" etc. Yes, this makes 100% sense.